packages/bloc/example/main.dart

```diff
@@ -22,7 +22,7 @@ class SimpleBlocObserver extends BlocObserver {
   }
 
   @override
-  void onError(Cubit cubit, Object error, StackTrace stackTrace) {
+  void onError(Cubit cubit, Object error, StackTrace? stackTrace) {
```

Comment:
I believe that generally stack traces are guaranteed to be non-null now (at least in the core apis) - although I don't know if there are `bloc` specific apis that don't guarantee that and would have to etc? It might be worth trying to force this non-nullable and see how that shakes out.

Reply:
good point 👍